js/src/personal/bryan/TimeDisplay.tsx

```diff
@@ -0,0 +1,70 @@
+import { useMemo, useState, useEffect } from 'react'
+import { Title } from '@mantine/core'
+import styles from './Bryan.module.css'
+
+export function TimeDisplay() {
+  const months = [
+    'Jan.',
+    'Feb.',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'Aug.',
+    'Sept',
+    'Oct.',
+    'Nov.',
+    'Dec',
+  ]
+  const [date, setDate] = useState(new Date())
+
+  useEffect(() => {
```

Comment:
nit: I'm a fan of adding comments explaining what useEffects do.
Generally useEffects need to be looked at with more scrutiny.